constants.py

```diff
@@ -54,3 +56,21 @@ class TraceData:
         # never None
         VARTYPE: pd.StringDtype(),
     }
+
+    TYPE_HINT_SCHEMA = {
+        FILENAME: pd.StringDtype(),
+        CLASS: pd.StringDtype(),
+        FUNCNAME: pd.StringDtype(),
+        LINENO: pd.UInt64Dtype(),
+        CATEGORY: pd.Int64Dtype(),
+        VARNAME: pd.StringDtype(),
+        VARTYPE: pd.StringDtype(),
+    }
+
+    METRICS_SCHEMA = {
+        FILENAME: pd.StringDtype(),
+        CLASS: pd.StringDtype(),
+        CATEGORY: pd.Int64Dtype(),
+        COMPLETENESS: pd.BooleanDtype(),
+        CORRECTNESS: pd.BooleanDtype(),
+    }
```

Comment:
I think we should store the compared symbols in here too, including the module path and the type name from BOTH sides, so that it is easier to see what differs

Reply:
What is the symbol name in this schema?

e.g. if I have `x: int = 5`, where are `x` and `int` stored?

Reply:
The metrics schema contains the columns for the metric data. The metric data is used to determine the correctness & completeness. I think you're thinking of the Type Hint Data.

Reply:
Yes, but how do you associate a metric with the compared symbol?

Reply:
You could perhaps store the index of the compared symbol in the Metrics DataFrame as a kind of foreign key into the parsed AST DataFrame

Reply:
After some thinking, the correctness & completeness is used to compare filter lists. However, the values of the columns are not causes for changes in correctness & completeness. Thus, causal inference should not occur. Due to that, only the correctness & completeness metric are computed and returned.

Reply:
Now, the overlap between two type hint data is considered as the metric data.

Reply:
Since line numbers might not match, indentation level is used instead.